internal.go

```diff
@@ -28,7 +28,10 @@ func makeGenesisDAG(daemon *docker.Client) error {
 	log15.Debug("created ethash container")
 	defer func() {
 		log15.Debug("deleting ethash container")
-		daemon.RemoveContainer(docker.RemoveContainerOptions{ID: ethash.ID, Force: true})
+		err := daemon.RemoveContainer(docker.RemoveContainerOptions{ID: ethash.ID, Force: true})
+		if err != nil {
+			log15.Error("failed to delete ethash container ", "error", err)
```

Comment:
I didn't add the IDs here or in shell.go because AIUI there's only one container for ethash/shell, and none of the logs emitted here/there include the ID. Should I do so anyway?